include/mrdox/Metadata/Symbols.hpp

```diff
@@ -25,7 +25,80 @@ namespace mrdox {
     USR. A USRs is a string that provide an
     unambiguous reference to a symbol.
 */
-using SymbolID = std::array<unsigned char, 20>;
+class SymbolID
+{
+    std::uint8_t data_[20];
+
+public:
+    constexpr SymbolID() = default;
+
+    template<typename Elem>
+    SymbolID(const Elem* src)
```

Comment:
I mean this is a little scary but fuck it, this ain't Boost :)

Reply:
LOL yeah